src/api/OpenWith.js

```diff
@@ -48,13 +54,38 @@ class OpenWith extends Base {
             id: fileId,
             params,
             successCallback: openWithIntegrations => {
-                const formattedOpenWithData = this.formatOpenWithData(openWithIntegrations);
+                const consolidatedOpenWithIntegrations = this.consolidateBoxEditIntegrations(openWithIntegrations);
+                const formattedOpenWithData = this.formatOpenWithData(consolidatedOpenWithIntegrations);
                 successCallback(formattedOpenWithData);
             },
             errorCallback,
         });
     }
 
+    /**
+     * Removes the Box Edit SFC integration if the higher scoped Box Edit integration is present.
+     *
+     * @param {OpenWithAPI} openWithIntegrations - Open With integration items
+     * @return {OpenWithAPI} Open With Integrations with only one Box Edit integration
+     */
+    consolidateBoxEditIntegrations(openWithIntegrations: OpenWithAPI): OpenWithAPI {
+        let { items } = openWithIntegrations;
+        const boxEditIntegration = items.find(
+            integration => integration.app_integration.id === BOX_EDIT_INTEGRATION_ID,
+        );
+
+        if (boxEditIntegration) {
+            items = items.filter(integration => {
+                return integration.app_integration.id !== BOX_EDIT_SFC_INTEGRATION_ID;
+            });
+        }
+
+        return {
+            ...openWithIntegrations,
```

Comment:
Did you intend to spread the top level object into the same level as its child? This means that you'll have both the content of `items` and an `items` property in the result.